src/constants.json

```
@@ -0,0 +1,16 @@
+{
+    "random": "Check this box to enable random safety car events.",
+    "random_max_occ": "The maximum number of times a safety car event could potentially occur (not guaranteed); applies to only random safety cars.",
+    "random_prob": "The probability of a single safety car deployment, irrespective of the maximum random occurrences. A value of 0.1 indicates a 10% chance of a safety car being deployed.",
+    "stopped": "Check this box to enable a safety car event when cars are stopped on the track.",
+    "stopped_min": "Minimum number of stopped cars on the track required to deploy a safety car.",
+    "message": "Message to be displayed to the drivers.",
+    "off": "Check this box to enable a safety car event when cars go off track.",
```

```suggestion
    "off": "Check this box to enable safety car events when cars go off track.",
```
Same thing, just making sure the user knows more than one of these can potentially be thrown.